diacamma/entrypoint.d/sub_command/restore.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+function run {
+    pushd /var/lucterios2 > /dev/null
+    filename="/backups/${1}"
+    if [ -f ${filename} ]
+    then
+        /bin/bash launch_lucterios.sh archive --name=${CURRENT_ORGANISATION} --file=${filename}
```

Comment:
Sound like a typo : should be "restore" instead of "archive" ?

Reply:
you're right, copy paste issue ... that could explain why it didn't work while testing ... PEBKAC issue ... ^_^